### DISCUSSION TITLE
makefile errors

### DISCUSSION THREAD

**Discussion by MOELHADYYY** [Cyfrin/foundry-full-course-cu#1008] [Q&A]
i am running a stm32 project and while using make file to delete or build something that error pops up ($ make clean
C:\WinAVR-20100110\utils\bin\make.exe: Interrupt/Exception caught (code = 0xc000
00fd, addr = 0x4217b3)) anyine can help

**Comment by theirrationalone** [comment]
It seems like, you're using `bash` on WSL and trying to execute make cmds forcefully via make.exe....

Please install `make` for your `bash` WSL and then try again.

```bash
sudo apt install make
```

**Comment by Amio13** [comment]
![Screenshot_2023-10-19_155920](https://github.com/Cyfrin/foundry-full-course-f23/assets/117745343/ca0d2d05-d67d-40e4-82eb-d226bb1e3d81)
Unrelated..but what maybe the cause?

**Reply by theirrationalone** [reply]
This could also be a reason... 

You're omitting to install `jq`

Try this... 

```bash
sudo apt install jq
```

After installing `jq` please try again  and please let me know if this works...

**Reply by Amio13** [reply]
It worked! Thanks sir...but it wasn't seen in the video... can you tell me more about this jq and when to install it ....

**Reply by theirrationalone** [reply]
It's Just an abbreviation for `JSON QUERY` === `jq`

And from where?? We are trying to read our txs data in which format??? 

That's broadcast folder containing txs data in json format. 

Now you can hopefully better think of this utility called jq

;`)

**Reply by Amio13** [reply]
Thanks Chief
